app/konnect/org-management/audit-logging/verify-signatures.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Verify audit log signatures
+content_type: how-to
+badge: enterprise
+beta: true
+---
+
+{{site.konnect_short_name}} uses an [ED25519 signature](https://ed25519.cr.yp.to/) on the audit logs it produces.
+
+Audit logs can be exported in two different formats, CEF and JSON. 
+Calculating the signature is slightly different for these formats.
+
+## Verify a signature
+
+1. Retrieve the public key from the audit log JWKS endpoint:
```

Comment:
It was not clear to me initially, but perhaps engineering can confirm if it is standard. But the public key is the value found in the `"x"` attribute of the JWKS endpoint.

Reply:
Based on some googling around, that sounds right for ED25519 keys.